jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -39,6 +40,7 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
         if (!ApplicationManager.getApplication().isUnitTestMode) {
             CodeWhispererStatusBarManager.getInstance(project).updateWidget()
         }
+        ToolkitUpdateManager.getInstance()
```

Comment:
think we can move this line to `AwsSettings`?

Reply:
the auto-update logic needs to be run once in the beginning when the user starts the IDE, in AwsSettings I don't find such places, so I decided to put it in some postStartUpActivity which ended up here.